src/main/resources/i18n/messages.properties

```diff
@@ -0,0 +1,22 @@
+# Error page
```

Comment:
Ok. Should I also delete those ? :

- the dir 
- addInterceptors in java.io...config.LocaleConfiguration.java
- ".antMatchers("/i18n/**")" in MicroserviceSecurityConfiguration.java